lib/matplotlib/tests/test_axes.py

```diff
@@ -3764,21 +3764,24 @@ def test_specgram_freqs():
     ax23 = fig2.add_subplot(3, 1, 3)
 
     ax11.specgram(y, NFFT=NFFT, Fs=Fs, noverlap=noverlap,
-                  pad_to=pad_to, sides='default')
```

Comment:
just set rcParams["image.interpolation"] = "nearest" above in the function and be done once for all?  we reset the rcparams at the end of each test anyways.

Reply:
Ooops, that would have saved some typing.  OTOH, I've already done it, can't hurt to leave it in explicitly..

Reply:
It adds a bit of noise to the code :(
You could just do `git checkout HEAD~ lib/matplotlib/tests/test_axes.py` to undo these changes, but it's up to you.